src/main/java/com/googlecode/objectify/impl/WriteEngine.java

```diff
@@ -68,6 +68,8 @@ public <E> Result<Map<Key<E>, E>> save(Iterable<? extends E> entities) {
 				final EntityMetadata<E> metadata = factory().getMetadataForEntity(obj);
 				final FullEntity<?> translated = metadata.save(obj, ctx);
 				entityList.add(translated);
+				System.out.println("Input POJO = " + obj);
+				System.out.println("Translated Entity = " + translated);

```

Comment:
Remove print debugging.

Reply:
This was to illustrate what I think is wrong behavior - once I have affirmation either way, I'll remove this - leaving unresolved until then.